add-on/panels/js/roomStore.js

```diff
@@ -544,7 +550,8 @@ loop.store = loop.store || {};
      */
     openRoom: function(actionData) {
       loop.requestMulti(
-        ["Rooms:Open", actionData.roomToken],
+        ["OpenURL", TOC_URL + actionData.roomToken],
+        ["LoadSidebar", actionData.roomToken],
```

Comment:
@Standard8 then what if we move the LoadSidebar code into the LoopRooms#open? Since we cannot load a different room in the same window, there's no reason to have an exposed method. wdyt?